examples/src/examples/AfiInteractiveOptimizerEngineExample.java

```diff
@@ -59,6 +55,9 @@ public static void main(String[] args) throws InterruptedException, JsonProcessi
       afiOptimizerParam.setData(afiItem);
       
       ApiResponse<Object> response = apiInstance.postAndOptimizeWithHttpInfo(null, null, afiOptimizerParam);
+      // Comment the above line and uncomment the below lines to add cache control configuration. Results are by default cached for 12 hours; Setting max-stale=300 will fetch a cached result which is 5 minutes older.
```

Comment:
Would it be better if we make it "Setting max-stale=300 will fetch a cached result which is at max 5 minutes older"?

Reply:
Yes, It's working fine for 5 minutes.

Reply:
@vnaredlawa Can you please make the change as suggested by @AkankshaAcharya(Include the word `max`) ?

Reply:
Changed the configuration lines and updated the code adding "at max" to all the products.